### PR TITLE
Major 03 2022

### PR DESCRIPTION
**Pull request by kevinkowa**
### Summary

Major release of the SDK Q1

### Models
* OutputData: BREAKING required text property removed
* RuntimeEntity: BREAKING optional metadata property removed
* RuntimeResponseGeneric: Three new response types added
* Workspace:  workspaceID changed form required to optional

### Models
* MessageContext: integrations property added
* MessageContextGlobalSystem: skipUserInput property added
* MessageContextStateless: integrations property added
* MessageInput: attachments property  added
* MessageInputStateless: attachments property  added
* MessageOutputDebug: BREAKING nodesVisited type DialogNodesVisited changed to DialogNodeVisited. NOTE: The DialogNodesVisited model are deleted
* RuntimeEntity: BREAKING optional metadata property removed
* RuntimeResponseGeneric: Three new response types added

### Models
* DocumentStatus: documentID, status, and statusDescription are now optional
* QueryAggregation: BREAKING QueryAggregation subclasses changed. Old models are deleted!
* 
## CC
* Removed

## LT
* Updated unit tests

## NLC
* Removed

## NLU
* Updated unit tests and comments

## PI
* Removed

### Models
* SupportedFeatures: customAcousticModel property added

## TTS
* Nothing

## Tone Analyzer
* Removed

## VisualRecognition
* Removed

### CODE REVIEW

File: examples/src/main/java/com/ibm/watson/assistant/v1/tone_analyzer_integration/AssistantToneAnalyzerIntegrationExample.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2019, 2020.
```
Comment (apaparazzi0329) [inline examples/src/main/java/com/ibm/watson/assistant/v1/tone_analyzer_integration/AssistantToneAnalyzerIntegrationExample.java:2]: This file should be deleted

File: examples/src/main/java/com/ibm/watson/assistant/v1/tone_analyzer_integration/ToneDetection.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2018, 2020.
```
Comment (apaparazzi0329) [inline examples/src/main/java/com/ibm/watson/assistant/v1/tone_analyzer_integration/ToneDetection.java:2]: File should be deleted

**Review by apaparazzi0329 — CHANGES_REQUESTED** [review]
LGTM

**Review by Hsaylor — APPROVED** [review]
straight up this PR is lagging chrome

**Review by repjarms — APPROVED** [review]

**Pull request merged**